Instruments/source/CacheTupleProducer.cxx

```diff
@@ -0,0 +1,161 @@
+/*! Produce synchronization tree.
+This file is part of https://github.com/hh-italian-group/h-tautau. */
+
+#include "AnalysisTools/Run/include/program_main.h"
+#include "AnalysisTools/Core/include/RootExt.h"
+#include "AnalysisTools/Core/include/EnumNameMap.h"
+#include "AnalysisTools/Core/include/AnalysisMath.h"
+#include "AnalysisTools/Core/include/TextIO.h"
+#include "h-tautau/Analysis/include/EventInfo.h"
+#include "h-tautau/Core/include/AnalysisTypes.h"
+#include "h-tautau/Cuts/include/Btag_2016.h"
+#include "h-tautau/Cuts/include/Btag_2017.h"
+#include "h-tautau/McCorrections/include/EventWeights.h"
+#include "hh-bbtautau/Analysis/include/SampleDescriptorConfigEntryReader.h"
+#include "h-tautau/Core/include/CacheTuple.h"
+#include "h-tautau/Core/include/EventTuple.h"
+#include "h-tautau/Analysis/include/SignalObjectSelector.h"
+
+struct Arguments {
+    REQ_ARG(std::string, input_file);
+    REQ_ARG(std::string, output_file);
+    REQ_ARG(std::string, tree_name);
+    REQ_ARG(std::string, trigger_cfg);
```

Comment:
cache producer should not depend on triggers. Remove here and all trigger-related parts below